contracts/peaks/yearn/YVaultPeak.sol

```diff
@@ -119,26 +119,15 @@ contract YVaultPeak is OwnableProxy, Initializable, IPeak {
     function redeemInYusd(uint dusdAmount, uint minOut) external {
         core.redeem(dusdAmount, msg.sender);
         uint r = dusdAmount.mul(1e18).div(yUSDToUsd()).mul(redeemMultiplier).div(MAX);
-        // there should be no reason that this contracts has yUSD, however being safe doesn't hurt
-        uint peakYusd = yUSD.balanceOf(address(this));
-        if (peakYusd < r) {
-            controller.withdraw(yUSD, r.sub(peakYusd));
-            uint peakYusd = yUSD.balanceOf(address(this));
-            // Insufficient yUSD. Convert yCRV => yUSD.
-            if (peakYusd < r) {
-                uint amount = r.sub(peakYusd);
-                uint peakYcrv = yCrv.balanceOf(address(this));
-                if (peakYcrv >= amount) {
-                    yCrv.safeTransfer(address(controller), amount);
-                    controller.earn(address(yCrv));
-                    controller.withdraw(yUSD, amount);
-                }
-                else {
-                    yCrv.safeTransfer(address(controller), peakYcrv);
-                    controller.earn(address(yCrv));
-                    controller.withdraw(yUSD, peakYcrv);
```

Comment:
When you deposit `peakYcrv` amount of yCRV you wouldn't receive `peakYcrv` amount of yUSD. This transfer will fail.